src/screens/Chat/GroupChatDetailScreen.js

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import { Alert, StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { GiftedChat } from "react-native-gifted-chat";
+import { firebase } from "../../firebase/Config";
+import { sendGroupChat } from "../../firebase/HandleChat";
+import FetchUserInfo from "../../firebase/FetchUserInfo";
+import FetchGroupChat from "../../firebase/FetchGroupChat";
+
+const initialState = {
+  username: "",
+  img: ""
+};
+
+const defaultAvatarUrl =
+  "https://firebasestorage.googleapis.com/v0/b/chattoku-e2672.appspot.com/" +
+  "o/default-profile.png?alt=media&token=e05118be-75e9-42ed-8568-c42a3d34d6f1";
```

Comment:
the defaultAvatarUrl is used in a lot of components. It might be better to put it in a seperate file